### DISCUSSION TITLE
user_dict สำหรับ spell check โดยที่คำมีตัว ขีด - หรือเครื่องหมาย + ด้วย

### DISCUSSION THREAD

**Discussion by thetorque** [PyThaiNLP/pythainlp#612] [Q&A] [answered]
สวัสดีครับ ผมอยากจะใช้ spellcheck กับลิสต์ของคำที่ผมกำหนดเอง แต่ปรากฏว่าคำที่ผมใช้อ้างอิงมันมี - ด้วย เช่น

สมคิด-สมศักดิ์

ปรากฎว่ามันไม่สามารถตรวจสอบคำสะกดที่มีตัว - ได้ เช่นทดสอบว่า checker.known(['สมคิด-สมศักดิ์]) มันก็บอกว่าไม่มีทั้ง ๆ ที่พิมพ์เหมือนกันเป๊ะ ๆ 

ไม่ทราบว่ามีวิธีแก้ไหมครับ

ขอบคุณมากครับ

**Comment by wannaphong** [comment, marked as answer]
ให้เพิ่มคำที่ถูกต้องเข้าไปที่ dict ชื่อ words_th.txt ครับ อยู่ที่ pythainlp/corpus/words_th.txt ครับ https://github.com/PyThaiNLP/pythainlp/blob/dev/pythainlp/corpus/words_th.txt
